.gitignore

```diff
@@ -13,3 +13,4 @@
 /frontend/npm-debug.log
 /frontend/yarn-error.log
 /frontend/gui_test_screenshots
+*.DS_Store
```

Comment:
can you move this near the top of the file before the directories?

Reply:
can do 👍